Formula/swift-format.rb

```diff
@@ -23,14 +23,19 @@ class SwiftFormat < Formula
   # out of the box on Xcode-only systems due to an incorrect sysroot.
   pour_bottle? only_if: :clt_installed
 
-  depends_on xcode: ["13.3", :build]
+  depends_on xcode: ["14.0", :build]
 
   uses_from_macos "swift"
 
   def install
     system "swift", "build", "--disable-sandbox", "-c", "release"
     bin.install ".build/release/swift-format"
     doc.install "Documentation/Configuration.md"
+
+    if OS.mac?
+      deuniversalize_machos ".build/release/lib_InternalSwiftSyntaxParser.dylib"
```

Comment:
Let's just skip the universal build by passing `-arch` to `swift build` instead of building a universal binary and thinning it afterward. (Or maybe it's `--arch` with two `-`s.)

Reply:
Oh, wait, we do need this since this library is likely copied from Xcode.

Reply:
It's actually copied in as an artifact from swift-syntax. They package it as a universal dylib.